helm/values.yaml

```diff
@@ -216,3 +216,5 @@ env:
   SECRET_KEY: "SpAA3v4icREtlC2ND3oZ8JABMEP2Cf1U"
   ALLOWED_HOSTS: "localhost,127.0.0.1"
   WAGTAILADMIN_BASE_URL: "http://localhost:8080/admin"
+  CSRF_TRUSTED_ORIGINS: "http://localhost:8080/,"
```

Comment:
No trailing comma unless needed

Reply:
I've added another label to remind that this is comma-separated.